### ISSUE TITLE
Break out Missing and DKR Living Situation Issue into more granular issues in DataQuality.R

### ISSUE THREAD

**Issue by kiadso** [kiadso/COHHIO_HMIS#17]
One reason is the user will find it more helpful
Another reason is in the CoC-wide Data Quality data, that error accounts for a LOT of errors and kind of obscures other (more granular) issues.

**Comment by kiadso** [comment]
Done. Waiting on WS to give me more information about LOSUnderThreshold works in the WS HUD CSV Export before I can make the final one live, but the rest are done.

**Comment by kiadso** [comment]
Closing this one out, will start a new issue about the LOSUnderThreshold.
